### DISCUSSION TITLE
GPax Discussions

### DISCUSSION THREAD

**Discussion by ziatdinovmax** [ziatdinovmax/gpax#54] [Announcements]
The Discussions are open now! 👋

**Comment by kevinroccapriore** [comment]
first!  :sunglasses:
...I've... never been first.. what a time to be alive!

**Comment by yongtaoliu** [comment]
Hi GPax!

**Comment by aghosh92** [comment]
Hi!

Big thumbs up for making road to learn about GPs much easier and informative, at the same time. :)
Suggestion: At some point, if possible, it might be a great to organize a symposium focused only on GPax itself and have some tutorials to make it more popular. I think it'll attract significant interests both from the lab and from outside. 
Would be happy to help.
--Ayana
